### PR TITLE
update NEWS file.

### PR DESCRIPTION
**Pull request by MrAE**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+
+* Urerf now supports using the Bayesian information criterion (BIC) from
+  the `mclust` package for determining the best split.
+
+* Feature importance calculations now correctly handle features whose
+  weight vectors parametrize the same line.  Also, when the projection
+  weights are continuous we tabulate how many times a unique combination
+  of features was used, ignoring the weights.
+
+* The base functionality is being moved into C++.
```
Comment (falkben) [inline NEWS.md]: added c++ implementation of randomer forest

File: NEWS.md [hunk 2]
```diff
@@ -12,7 +12,7 @@
   weights are continuous we tabulate how many times a unique combination
   of features was used, ignoring the weights.
 
-* The base functionality is being moved into C++.
+* A C++ implmentation of Randomer Forest has been added.
```
Comment (falkben) [inline NEWS.md]: spelling

Comment (falkben) [reply NEWS.md]: @MrAE implmentation -> implementation

File: NEWS.md [hunk 3]
```diff
@@ -1,12 +1,29 @@
-Changes in 2.0.2:
+## Changes in 2.0.3:
```
Comment (falkben) [inline NEWS.md]: maybe double check here to make sure we are getting everything?

https://github.com/neurodata/RerF/compare/master...staging

Comment (falkben) [reply NEWS.md]: do we include our bugfixes here?  like the issue with splitting on an empty node?

**Comment by MrAE** [comment]
Moved updates to the NEWS file to the branch `jlp/cranV2.0.3`

**Pull request closed without merging**